File: framework/src/main/kotlin/br/com/zup/beagleui/framework/widget/ui/Image.kt
package br.com.zup.beagleui.framework.widget.ui

import br.com.zup.beagleui.framework.widget.core.ImageContentMode
import br.com.zup.beagleui.framework.widget.core.NativeWidget

data class Image(
    val name: String,
    val contentMode: ImageContentMode? = null /* = ImageContentMode.FIT_CENTER */
) : NativeWidget

File: Sources/BeagleUI/Sources/DeclarativeViews/UIComponents/Image/Image.swift
//
//  Image.swift
//  BeagleUI
//
//  Created by Daniel Tes on 12/09/19.
//  Copyright © 2019 Daniel Tes. All rights reserved.
//

public struct Image: NativeWidget {
    
    // MARK: - Public Properties
    
    public let name: String
    private let contentMode: ImageContentMode
    
    // MARK: - Initialization
    
    public init(
        name: String,
        contentMode: ImageContentMode = .fitCenter
    ) {
        self.name = name
        self.contentMode = contentMode
    }
    
}
